core/network/src/test/sync.rs

```diff
@@ -67,13 +67,15 @@ fn sync_no_common_longer_chain_fails() {
 
 #[test]
 fn sync_justifications() {
-	::env_logger::init().ok();
+	let _ = ::env_logger::try_init();
 	let mut net = JustificationTestNet::new(3);
 	net.peer(0).push_blocks(20, false);
 	net.sync();
 
 	// there's currently no justification for block #10
 	assert_eq!(net.peer(0).client().justification(&BlockId::Number(10)).unwrap(), None);
+	assert_eq!(net.peer(1).client().justification(&BlockId::Number(10)).unwrap(), None);
```

Comment:
Had an issue here and was debugging this, added the line and left it in, because it is a good check.